README.md

```diff
@@ -281,6 +281,15 @@ var httpClient = new HttpClient(new HttpClientXRayTracingHandler(new HttpClientH
 httpClient.GetAsync(URL);
 ```
 
+Using the [HttpClientFactory](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests) pattern
+
+The `Amazon.XRay.Recorder.Handlers.System.Net` package includes a delegate that can be used to trace outbound requests without the need to specifically wrap outbound requests from that class.
+
+```csharp
+services.AddHttpClient<IAmazonClient, AmazonClient>()
```

Comment:
I'm unable to understand which package the IAmazonClient and AmazonClient belong to. If you can add more detail to the example, it would be better.

Reply:
This was an arbitrary example (but I was trying to keep it on Amazon-trend), I'll move it to IFooClient and FooClient to make it more obvious.